osm2gtfs.py

```diff
@@ -298,5 +293,34 @@ def load_config(file):
 
     return config;
 
+def load_agency(config):
+    '''
+    Loads agency data from a json config file. The valid keys under the json
+    "agency" object correspond to the transitfeed.Agency._FIELD_NAMES.
+
+    return a transitfeed.Agency object
+'''
+
+    data_dict = {}
+    for field_name in transitfeed.Agency._FIELD_NAMES:
+        try:
+
+            field_value = config["agency"][field_name]
+            if field_value is not None and field_value != "":
+                data_dict[field_name] = field_value
+
+        except KeyError:
+            sys.stderr.write("key '"+field_name+"' was not found on the config file.")
+
+    agency = transitfeed.Agency(field_dict=data_dict)
+
+    if not agency.Validate():
+        sys.stderr.write("Agency data has errors")
+        # later, an error handling based on a exception here could be added.
+        #raise AttributeError('Agency data on config file in not valid.')
```

Comment:
The validate method returns a boolean and have a mechanism to handle problems. I think the method "load_agency" could be improved to be less naive.